packages/core/src/plugins/manifest.ts

```diff
@@ -163,6 +163,7 @@ export const pluginManifest = (): RsbuildPlugin => ({
       chain.plugin(CHAIN_ID.PLUGIN.MANIFEST).use(RspackManifestPlugin, [
         {
           fileName,
+          writeToFileEmit: true,
```

Comment:
Will this option conflict with Rspack's built-in emit? Should we enable this only in dev mode

Reply:
There will be no conflict, but it is better to avoid repeated emit.